india_compliance/gst_india/report/isd_invoice_register/isd_invoice_register.js

```
@@ -0,0 +1,166 @@
+};
+
+function custom_report_column_total(...args) {
+    const column = args[1].column;
+    const column_field = column.fieldname;
+    if (![...AMOUNT_FIELDS, "total_invoice_value", "taxable_value"].includes(column_field)) return "";
+
+    const { data } = this.datamanager;
+    const indices = this.datamanager.getFilteredRowIndices();
+
+    // base_grand_total repeats across the per-rate rows of an invoice; count it once per invoice.
+    if (column_field === "total_invoice_value" || column_field == "taxable_value") {
```

Biome (2.5.0)

[error] Using == may be unsafe if you are relying on type coercion.

(lint/suspicious/noDoubleEquals)